address-book.json

```diff
@@ -224,20 +224,23 @@
     "TwoPartyFixedOutcomeFromVirtualAppInterpreter": {
       "address": "0xf204a4Ef082f5c04bB89F7D5E6568B796096735a"
     },
-    "SimpleTransferApp": {
+    "SimpleLinkedTransferApp": {
       "address": "0x75c35C980C0d37ef46DF04d31A140b65503c0eEd"
     },
-    "SimpleTwoPartySwapApp": {
+    "SimpleTransferApp": {
       "address": "0x82D50AD3C1091866E258Fd0f1a7cC9674609D254"
     },
-    "UnidirectionalLinkedTransferApp": {
+    "SimpleTwoPartySwapApp": {
       "address": "0xdDA6327139485221633A1FcD65f4aC932E60A2e1"
     },
-    "UnidirectionalTransferApp": {
+    "UnidirectionalLinkedTransferApp": {
```

Comment:
Are these UnidirectionalApp references on the chopping block too? Should we remove them from the contracts folder too?